pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1330,7 +1330,8 @@ func TestSelectNodesForPreemption(t *testing.T) {
 			newnode := makeNode("newnode", 1000*5, priorityutil.DefaultMemoryRequest*5)
 			newnode.ObjectMeta.Labels = map[string]string{"hostname": "newnode"}
 			nodes = append(nodes, newnode)
-			nodeToPods, err := selectNodesForPreemption(test.pod, nodeNameToInfo, nodes, test.predicates, PredicateMetadata, nil, nil)
+			pluginContext := framework.NewPluginContext()
+			nodeToPods, err := selectNodesForPreemption(pluginContext, test.pod, nodeNameToInfo, nodes, test.predicates, PredicateMetadata, nil, nil, func(pc *framework.PluginContext, pod *v1.Pod, nodeName string) *framework.Status { return nil })
```

Comment:
Can we add a test for what we are adding? preemption when the filter plugins keep filtering the nodes.

Reply:
Did you work on this?

Reply:
Sorry, I missed it. If I am not mistaken, the test case you are talking about is actually to confirm that the filter plugins are also executed when preemption occurs.

This does not seem to be a test case, but rather the number of times the filter plugins are executed.

I will check filterPlugin.numFilterCalled with the expected in TestSelectNodesForPreemption. What do you think of this?

Reply:
that sounds good. But you could also check that a plugin responds Unschedulable in the preemption phase and then the node is not returned in the list.

Reply:
DONE